packages/application-shell-connectors/src/components/project-extension-image-regex/project-extension-image-regex.tsx

```diff
@@ -79,9 +73,15 @@ function withProjectExtensionImageRegex<Props extends {}>(
   };
 }
 
+const useProjectExtensionImageRegex = () => {
```

Comment:
I think this hook should be also documented in `application-shell-connectors` docs. 

Please also remember about the changeset 🤗

Reply:
I agree with @kark, we need to update [the docs](https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-application-shell-connectors#project-image-settings) to include the new hook and to explained that the reder-prop component and the HOC are deprecated.

We can do it in a follow-up PR if you don't want to extend this one.

Reply:
I will raise a separate follow up PR for this?

Reply:
It's ok for me 👍